src/cluster_legacy.c

```diff
@@ -3629,7 +3629,7 @@ int clusterProcessPacket(clusterLink *link) {
             serverLog(
                 LL_NOTICE,
                 "Closing link for node %.40s that sent a lightweight message of type %s as its first message on the link",
-                hdr->sender,
+                link->node ? link->node->name : "",
```

Comment:
Shall we display some placeholder instead of just blank node name?

```suggestion
                link->node ? link->node->name : "(unknown)",
```

Reply:
I saw it being empty somewhere else. But I would prefer unknown as well.

Reply:
In most places it is ```"<unknown>"``` and sometimes it is ```"NULL"``` or just empty "".
Maybe some function is needed to get node names consistent?

Reply:
@VyacheslavVanin Yeah, make sense. Updated.